static/js/copy_and_paste.js

```diff
@@ -318,6 +328,23 @@ export function paste_handler(event) {
     }
 
     if (clipboardData.getData) {
+        const paste_text = clipboardData.getData("text");
+        if (paste_text && validURL(paste_text)) {
+            event.preventDefault();
+            event.stopPropagation();
+
+            const textarea = $(this);
+            const prefix = "[";
+            const suffix = "](" + paste_text + ")";
+            const range = textarea.range();
```

Comment:
I don't see this checking whether there is a range selected using e.g. `document.getSelection().type === "Range"` before we decide whether to do this different behavior.  Is that intentional?  I would think that if we didn't have anything selected, we'd want pasting a link to just paste the link.

In any case, it'd be good to have tests.

Reply:
Sure, I thought adding this condition wasn't much needed.

Reply:
Well, it creates a link inside `[]()`, which the user may not want.

Reply:
Yep changed that behavior.